apps/prairielearn/elements/pl-variable-score/pl-variable-score.py

```diff
@@ -44,23 +44,23 @@ def render(element_html, data):
         html = (
             '<span class="badge badge-success">'
             + '<i class="fa fa-check" aria-hidden="true"></i>'
-            + (" correct: %d%%" % math.floor(score * 100))
+            + f" correct: {math.floor(score * 100):d}%"
```

Comment:
I somewhat blindly added the same format specifier (`:d`), but maybe we don't need that?

Reply:
As pointed out by someone in the other PR, unless you need to explicitly convert to a type for some reason, the default should just work.

Reply:
And for context, I would never write code with `{:d}`on something that is already an integer.

Reply:
Removed in 9bd62df.